flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/utils/TestingLogicalSlotProvider.java

```diff
@@ -53,30 +52,31 @@
 /**
  * A testing utility slot provider that simply has a predefined pool of slots.
  */
-public class SimpleSlotProvider implements SlotProvider, SlotOwner {
+public class TestingLogicalSlotProvider implements SlotProvider, SlotOwner {
```

Comment:
Also, here, is this renaming needed? At least, it could be shorter `TestingSlotProvider` because `SlotProvider` interface is already about `LogicalSlot`s.

Reply:
There are many `TestingSlotProvider` in fact...Also we can get few info from the name `SimpleSlotProvider`.

Reply:
Having `LogicalSlot` also does not seem to add any info to its name, all `SlotProvider`s provide `LogicalSlot`s. In this case, having only one `SimpleSlotProvider` is actually more clear because it says that it is the most obvious implementation of `SlotProvider`.